packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/hierarchy/HierarchyLifecycleControllerTests.java

```diff
@@ -76,7 +77,7 @@ public void givenAnArbitrarySemanticElementThenWeCanCreateHierarchyOnIt() {
                 .map(HierarchyRefreshedEventPayload.class::cast)
                 .map(HierarchyRefreshedEventPayload::hierarchy)
                 .ifPresentOrElse(hierarchy -> {
-                    assertThat(hierarchy.getChildNodes()).isEmpty();
+                    assertFalse(hierarchy.getChildNodes().isEmpty());
```

Comment:
This PR has been failing because of an invalid Git commit message but I suspect that this has hidden the fact that it cannot work on master today (we could find from which commit it may have worked if it has even worked one day). In order to make it work, this test should be valid (i.e. successfully creating a hierarchy with at least one node). But for that it should create a valid node.

For that it should create a valid `HierarchyNodeElementProps` which include having a valid label (since we have a Objects.requireNonNull in the code). But the hierarchy description has one flaw, it contains a label provider used to compute both the name of the representation and the name of the nodes. In the case of this test, the description has the following behavior:

```
        Function<VariableManager, String> labelProvider = variableManager -> variableManager.get(HierarchyDescription.LABEL, String.class)
                .orElse(null);
```

Which means that it will wait for the variable "label" used when creating a new representation otherwise it will return `null`. Given the fact that the only variables available today are:

```
        VariableManager variableManager = new VariableManager();
        variableManager.put(VariableManager.SELF, targetObject);
        variableManager.put(IEditingContext.EDITING_CONTEXT, editingContext);
```

and

```
        for (Object semanticElement : semanticElements) {
            VariableManager childVariableManager = variableManager.createChild();
            childVariableManager.put(VariableManager.SELF, semanticElement);



            String label = hierarchyDescription.getLabelProvider().apply(childVariableManager);
        }
```

in the condition of the test, this PR cannot possibly create a valid hierarchy now that the test data of the ecore project have been modified.